src/System.Threading.Tasks.Channels/System/IO/Channels/Channel.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.Channels
+{
+    /// <summary>Provides static methods for creating channels.</summary>
+    public static class Channel
+    {
+        /// <summary>Creates an unbounded channel usable by any number of readers and writers concurrently.</summary>
+        /// <returns>The created channel.</returns>
+        public static Channel<T> CreateUnbounded<T>() =>
```

Comment:
I still wish we chose one channel type as a default and called the factory method simply "Create"

Reply:
There wasn't any agreement on what that default type would be.  For users coming from Go, the default there is "Unbuffered".  In terms of usefulness, I expect "Unbounded" is the most useful (it's the direct equivalent of List/Queue/Dictionary/etc. which are all "unbounded").  But devs wanting to avoid OOM issues might want the default to be "Bounded".  Seems like the best option is to force the dev to make a decision about this up-front and to make the choice obvious.